unlock-app/.babelrc

```diff
@@ -38,7 +40,8 @@
             "preset-env": {
               "modules": "commonjs"
             }
-          }
+          },
+          "@zeit/next-typescript/babel"
```

Comment:
I'm not 100% sure why, but the order of the plugins mattered on this environment. Putting `next-typescript` in as the second element was an error.

All of these are added so that we can build typescript files. This was a pretty easy change--after this config all the building scripts worked exactly as they did before.